app/views/admin/users/_form.html.slim

```diff
@@ -0,0 +1,11 @@
+= simple_form_for @user, as: :user, url: url do |f|
+  h4 = title
```

Comment:
the title can be placed into template edit/new. this is not a mistake. just a little detail